gateway/mw_go_plugin.go

```diff
@@ -63,7 +63,9 @@ func (w *customResponseWriter) WriteHeader(statusCode int) {
 }
 
 func (w *customResponseWriter) Flush() {
-        w.ResponseWriter.(http.Flusher).Flush()
+       if _, ok := w.ResponseWriter.(http.Flusher); ok {
```

Comment:
I feel like you can do `flusher` instead of `_` and then just use flusher.Flush() without a double cast. :pray:

Reply:
Hi @titpetric Can this be approved please?

Reply:
I need to do some internal housekeeping to include it in the next sprint. It's gonna be some delay, but unless some new concerns rise up, it's a merge candidate. 👍

Reply:
Thank you!